src/Message/SendMessageHandler.php

```diff
@@ -24,9 +25,9 @@ public function __invoke(SendMessage $sendMessage): void
         $message->setUuid(Uuid::v6()->toRfc4122());
         $message->setText($sendMessage->text);
         $message->setStatus('sent');
-        $message->setCreatedAt(new \DateTime());
+        $message->setCreatedAt(new DateTime());
```

Comment:
This call is not needed because `createdAt` is populated automatically in the constructor. But I left it anyway; namespace removal is automatic onSave action and makes code cleaner.